src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/ConvertToJsonCommand.cs

```diff
@@ -99,23 +99,7 @@ protected virtual void Dispose(bool disposing)
                 _cancellationSource.Dispose();
             }
         }
-
-        /// <summary>
-        /// Prerequisite checks.
-        /// </summary>
-        protected override void BeginProcessing()
-        {
-            if (_depth > maxDepthAllowed)
```

Comment:
The variable `maxDepthAllowed` should be removed too.

Reply:
maxDepthAllowed removed.